front/app/services/formCustomFields.ts

```diff
@@ -171,3 +171,45 @@ export function formCustomFieldsResultsStream(
     ...streamParams,
   });
 }
+
+export interface IFormSubmissionCountData {
+  totalSubmissions: string;
+}
+
+export interface IFormSubmissionCount {
+  data: IFormSubmissionCountData;
+}
+
+export function formSubmissionCountStream(
```

Comment:
I moved this here such that all the work handling this is in one place. It makes it easy to work on similar changes in one